htmltools/_core.py

```diff
@@ -1238,27 +1238,19 @@ def _static_extract_serialized_html_deps(
 # =============================================================================
 # HTML strings
 # =============================================================================
-class HTML(str):
-    """
-    Mark a string as raw HTML. This will prevent the string from being escaped when
-    rendered inside an HTML tag.
 
-    Examples
-    --------
-    >>> from htmltools import HTML, div
-    >>> div("<p>Hello</p>")
-    <div>&lt;p&gt;Hello&lt;/p&gt;</div>
-    >>> div(HTML("<p>Hello</p>"))
-    <div><p>Hello</p></div>
+
+class RAW(str):
```

Comment:
This isn't an acronym, is it? I would've expected `Raw`

Reply:
Oh yes of course, changed